proposals/lip-0071.md

```diff
@@ -86,8 +89,8 @@ We calculate the rewards for validators as follows:
 | Name | Mainchain Value | Computation | Description |
 |------|-----------------|-------------|-------------|
 | `minimumRewardActiveValidators` | 10<sup>7</sup> | (`FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS * getDefaultRewardAtHeight(blockHeader.height))//DECIMAL_PERCENT_FACTOR` | The minimum reward per round for an active validator. Here `DECIMAL_PERCENT_FACTOR == 10000` and `blockHeader.height` is the current block height. |
-| `totalRewardActiveValidators` | 101*10<sup>8</sup> | `NUMBER_ACTIVE_VALIDATORS * getDefaultRewardAtHeight(blockHeader.height)` | The total reward per round for active validators. Here `blockHeader.height` is the current block height. |
-| `stakeRewardActiveValidators`| 90.9*10<sup>8</sup> | `totalRewardActiveValidators - NUMBER_ACTIVE_VALIDATORS * minimumRewardActiveValidators` | The remaining rewards for active validators (per round) after giving the minimum reward. |
+| `totalRewardActiveValidators` | 101*10<sup>8</sup> | `Pos.getActiveValidatorsNumber * getDefaultRewardAtHeight(blockHeader.height)` | The total reward per round for active validators. Here `blockHeader.height` is the current block height. |
```

Comment:
We don't use a consistent spelling for the method calls to the PoS module. I would either use `PoS` or `pos` consistently in this LIP.

Reply:
Ok I guess this is because I edited a LIP which I did not write. I think the convention in LIPs is to capitalize the first letter of module name in pseudocode, so I would use `Pos.xxx`. 

Plz check it is now consistent.

Reply:
Looks good.